examples/multigpu_advtrain/evaluator.py

```diff
@@ -0,0 +1,333 @@
+import os
+import numpy as np
+import logging
+import warnings
+import math
+
+import tensorflow as tf
+
+from cleverhans.utils_tf import _FlagsWrapper
+from cleverhans.attacks import FastGradientMethod, BasicIterativeMethod
+
+from model import MLP_probs
+from target_attack import ProjectedGradientDescentMethod
+
+
+def model_eval(sess, x, y, predictions=None, X_test=None, Y_test=None,
```

Comment:
Is it necessary to copy-paste this `model_eval` function or could we re-use the one from the core library?

Reply:
The `model_eval` in the core library cannot handle a batch not divisible by batch_size. So this modification is good to have in the core `model_eval` is well. I made a new commit 2ef5a2c and changed the core `model_eval` and removed the one in this file.

Reply:
OK, thanks. For future work, it would be better to make the change to `model_eval` in a separate PR, so we could review one small PR updating `model_eval`. It gets difficult to review large PRs containing multiple changes.